editor.planx.uk/src/@planx/components/Feedback/model.ts

```diff
@@ -24,3 +27,21 @@ export const parseFeedback = (
   feedbackRequired: data?.feedbackRequired || defaultContent.feedbackRequired,
   ...parseBaseNodeData(data),
 });
+
+export const createFeedbackSchema = (feedbackRequired: boolean) => {
+  return object().shape({
+    userComment: feedbackRequired
+      ? string().required("Enter your feedback")
+      : string(),
+    feedbackScore: feedbackRequired
+      ? number()
+          .integer()
+          .min(1, "Feedback score must be at least 1")
+          .max(5, "Feedback score cannot exceed 5")
+          .required("Please provide a feedback score")
+      : number()
+          .integer()
+          .min(1, "Feedback score must be at least 1")
+          .max(5, "Feedback score cannot exceed 5"),
```

Comment:
nit: while this validation schema is totally correct, are `min` & `max` actually useful user-facing errors here since the user is interacting with emoji expressions rather than number scores? Basically I think I'd vote `.required()` or undefined is plenty here? 

Also perhaps `"Please rate your experience"` would better align to default label text rather than `"Please provide a feedback score"` ?